iroh-gateway/Cargo.toml

```diff
@@ -24,6 +24,7 @@ git-version = "0.3.5"
 rand = "0.8.5"
 tracing-opentelemetry = "0.17.2"
 opentelemetry = { version = "0.17.0", features = ["rt-tokio"] }
+chrono = "0.4.19"
```

Comment:
lets try and use time.rs instead, chrono has had some security issues in the past